1bash

```diff
@@ -364,25 +366,27 @@ OVERCLOCK_MODE="GLOBAL"     # NO, GLOBAL, GPU_SPECIFIC, ALGO_SPECIFIC
 
```

Comment:
@LuKePicci 
Can you please add info in 1bash for new OC settings?
I think there are missing info about the 2 new OFFSET modes...

Reply:
I remember our talks with leenoox, but I also think since we already have the codes, let's give pro users chance to use em, there are already people asking for those options in discord.
You can add a Pro OC settings so beginners dont mess with those...